### DISCUSSION TITLE
Ubuntu upgrade problem, get 1.2.4.dev0

### DISCUSSION THREAD

**Discussion by GSXswe** [Chia-Network/chia-blockchain#7840] [Support]
Hello,

Just upgraded my Ubuntu 20.04 Chia server, when I follow the instructions I get the release 1.2.4.dev0 installed, had 1.2.2 before. And 1.2.2 was upgraded from 1.2.0 before and it worked just fine that time.

I have tried the instruction:  # If you get RELEASE.dev0 then delete the package-lock.json in chia-blockchain-gui and install.sh again ,  but no luck...  Same version every time I try.

git log output: 

commit b593f55dcd35ef6ca48a4db0c3f57a46947da767 (HEAD -> latest, tag: 1.2.3, origin/latest)
Author: paulhainsworth-chia <86693700+paulhainsworth-chia@users.noreply.github.com>
Date:   Mon Jul 26 16:01:25 2021 -0700

    changelog for 1.2.3 (#7702)

    changelog for 1.2.3

    Changelog for 1.2.3 updated 7-26-21

**Comment by emlowe** [comment]
A small change to the instructions - run `git reset --hard FETCH_HEAD --recurse-submodule` after `git checkout latest`

**Comment by GSXswe** [comment]
Thanks, that did the trick.  :-)
